{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/templates/base/_analytics.html

```diff
@@ -13,6 +13,17 @@
     })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
 
     ga('create', '{{ settings.GOOGLE_ANALYTICS }}', 'auto');
+    // Only anonymize if not consented.
```

Comment:
Is there a reason we should not anonymise regardless of cookie consent?

Reply:
Also, when comments are in HTML, you should use template `{# comments #}` rather than `// comments` or `<!-- comments -->` - it is not useful to be in the rendered HTML.

Reply:
(I will update the style guide with this.)

Reply:
I assumed the clients would want more accurate analytics if they could but I can change this to always anonymise IP